packages/react/src/themes/types.ts

```diff
@@ -356,6 +378,8 @@ export interface ThemeComponentStylesPrepared {
   DropdownItem?: ComponentSlotStylesPrepared
   DropdownSearchInput?: ComponentSlotStylesPrepared
   Embed?: ComponentSlotStylesPrepared
+  Flex?: ComponentSlotStylesPrepared
```

Comment:
this is a clear indication that these types were useless in a sense of declaring each component individually, as nothing was checked/guaranteed with them. This entire declaration was identical, in a sense of its effect, to
```js
type ThemePrepared = { [K: string]: ComponentSlotStylesPrepared }
```